### PR TITLE
Add MetricSelector.js

### PR DESCRIPTION
**Pull request by ecomerford**
@chutten through this PR, I have done the following. 

1. add `MetricSelector.js` component.
2. add state (`metricQuery`) to the app.
3. create an input field that handles input by updating state.
4. standardize use of `""` versus `''` throughout.
5. add fluidity to bootstrap components to help with responsiveness.

Please review when you have a chance! 💯

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,9 +24,7 @@
     <title>React App</title>
   </head>
   <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
-    </noscript>
+    <script src="https://telemetry.mozilla.org/v2/telemetry.js"></script>
```
Comment (openjck) [inline public/index.html]: The `<noscript>` thing can be a good thing to keep around. The wrapped message only appears when someone has JavaScript disabled. Try disabling JavaScript in Firefox Developer Tools to see.

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+import {ViewSelector} from "./Components/ViewSelector.js";
+import {MetricSelector} from "./Components/MetricSelector.js";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    metricQuery: "",
+    }
+    this.handleInputChange = this.handleInputChange.bind(this);
```
Comment (openjck) [inline src/App.js]: There's a neat way around this that I recently discovered myself. You can omit this line if you define `handleInputChange` like this:

```javascript
handleInputChange (event) {
  this.setState({
  metricQuery: event.target.value
  })
  console.log(this.state.metricQuery);
}
```

See [this blog post section](https://medium.com/komenco/react-autobinding-2261a1092849#af31) for more info. As with everything, there are pros and cons with this, but if you find that you have a lot of `this.whatever = this.whatever.bind(this)` lines in your code, it's a trick worth keeping in your back pocket.

Comment (openjck) [inline src/App.js]: There's a neat way around this that I recently discovered myself. You can omit this line if you define `handleInputChange` like this:

```javascript
handleInputChange = event => {
  this.setState({
  metricQuery: event.target.value
  })
  console.log(this.state.metricQuery);
}
```

See [this blog post section](https://medium.com/komenco/react-autobinding-2261a1092849#af31) for more info. As with everything, there are pros and cons with this approach, but if you find that you have a lot of `this.whatever = this.whatever.bind(this)` lines in your code, it's a trick worth keeping in your back pocket.

**Comment by ecomerford** [comment]
Thanks @openjck! I've added a few commits to affect those changes. I didn't realize that arrow functions were binding, that's really good to know! :)

Also, we don't need the `telemetry.js` script, so I took that out.

**Comment by chutten** [comment]
Of course you can be burned the other way with `=>` syntax. I've fallen afoul of it with event handlers which sometimes put crucial information on their `this` value (like `XMLHttpRequest`), and using `=>` will bind the surrounding `this` which won't have the info you want.

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+import "./App.css";
+import {Navigation} from "./Components/Navigation.js";
+import {ViewSelector} from "./Components/ViewSelector.js";
+import {MetricSelector} from "./Components/MetricSelector.js";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    metricQuery: "",
```
Comment (chutten) [inline src/App.js]: Doesn't this need an additional level of indentation?

File: src/App.js [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
+    }
+  }
+
+  handleInputChange = (event) => {
+    this.setState({
+    metricQuery: event.target.value
```
Comment (chutten) [inline src/App.js]: indentation and defensive comma

**Review by chutten — APPROVED** [review]
A couple of things I'd have expected prettier to have caught, otherwise looks good.

**Comment by ecomerford** [comment]
Something to keep an eye out for, then! Thanks, @chutten! I'll go ahead and merge the PR now.

**Pull request merged**